### PR TITLE
Fix deployment of allocator to testnet

### PR DESCRIPTION
**Pull request by 0xJem**

### CODE REVIEW

File: scripts/deploy/009_deploy_allocator.ts [hunk 1]
```diff
@@ -7,35 +7,37 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
...
+            authorityDeployment.address,
             treasuryDeployment.address,
             lusdTokenAddress,
             lqtyToken,
             stabilityPool,
             stakingPool,
             "0x0000000000000000000000000000000000000000",
-            weth
+            weth,
+            "0x0000000000000000000000000000000000000000",
```
Comment (lolchocotaco) [inline scripts/deploy/009_deploy_allocator.ts:33]: So this is the hopTokenAddress, which we're using as DAI for now
`0x6b175474e89094c44da98b954eedeac495271d0f `

**Review by lolchocotaco — COMMENTED** [review]
One smol thing about `hopTokenAddress`

File: scripts/deploy/009_deploy_allocator.ts (hunk 1 above)
Comment (0xJem) [reply scripts/deploy/009_deploy_allocator.ts]: Done

**Comment by lolchocotaco** [comment]
IM DOWN.

**Review by lolchocotaco — APPROVED** [review]
Looks good to me.

**Pull request merged**